### PR TITLE
Allow adding custom classes to the suggest list and suggest item

### LINKED ISSUE #197: Support adding class names to suggests and suggest items
Is possible to set className for suggests and suggest items just as we set for inputClassName props?

This is my code

```
import React from 'react'
import classes from './LocationInputComponent.scss'
import Geosuggest from 'react-geosuggest';

export const LocationInputComponent = () => (
  <div>
    <Geosuggest
      inputClassName={ classes.geosuggest } />
  </div>
)
```

In the LocationInputComponent.scss file, I have exactly the same classes as [https://github.com/ubilabs/react-geosuggest/blob/master/src/geosuggest.css] 

The result of `classes.geosuggest` is something like this:  `geosuggest__input LocationInputComponent__geosuggest___91ULR`

I would like to be able to do something like this:

```
import React from 'react'
import classes from './LocationInputComponent.scss'
import Geosuggest from 'react-geosuggest';

export const LocationInputComponent = () => (
  <div>
    <Geosuggest
      inputClassName={ classes.geosuggest }
      suggestsClassName={ classes.geosuggest__suggests }
      suggestItemClassName={ classes.geosuggest__suggest__item } />
  </div>
)
```

I believe, this is the same (or almost the same) issue as: [https://github.com/ubilabs/react-geosuggest/issues/62]

### PR DESCRIPTION
**Pull request by karanjthakkar**
### Description

Fixes #197. Reuses the work done by @edudavid in https://github.com/ubilabs/react-geosuggest/pull/198

### Checklist

- [x] All tests passing
- [x] Extended the README / documentation, if necessary
- [x] Commits and PR follow conventions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,6 +92,18 @@ Default: `''`
 
 Add an additional class to the input.
 
+#### suggestListClassName
```
Comment (ro-ka) [inline README.md]: Can you name this `suggestsClassName` to be in line with the `style.suggests` and the `suggestsHiddenClassName` properties?

**Comment by ro-ka** [comment]
Would you mind to update the naming? Can merge this then.  😃

**Comment by karanjthakkar** [comment]
@ro-ka Sorry about the delay. I deleted the fork coz I was trying something out. I'll create a new PR on a new fork with the new name changes. I might need a bit more time for adding tests as I have a few other engagements that need attention. 

Happy new year! 🎆

**Comment by ro-ka** [comment]
No problem! Update as soon as you’re ready.   😃  🎆

**Comment by karanjthakkar** [comment]
Closing in favor of #262.

**Pull request closed without merging**